### PR TITLE
Elcomsoft.AdvancedPDFPasswordRecovery version 5.20.194.6989

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102164)

### CODE REVIEW

File: manifests/e/ElcomsoftCo/Ltd/AdvancedPDFPasswordRecovery/5.20.194.6989/ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/Ltd/AdvancedPDFPasswordRecovery/5.20.194.6989/ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery.installer.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.AdvancedPDFPasswordRecovery
```

File: manifests/e/ElcomsoftCo/Ltd/AdvancedPDFPasswordRecovery/5.20.194.6989/ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/Ltd/AdvancedPDFPasswordRecovery/5.20.194.6989/ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.AdvancedPDFPasswordRecovery
```

File: manifests/e/ElcomsoftCo/Ltd/AdvancedPDFPasswordRecovery/5.20.194.6989/ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/Ltd/AdvancedPDFPasswordRecovery/5.20.194.6989/ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.AdvancedPDFPasswordRecovery
```

File: manifests/e/ElcomsoftCo/Ltd/AdvancedPDFPasswordRecovery/5.20.194.6989/ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery
+PackageVersion: 5.20.194.6989
+PackageLocale: en-US
+Publisher: Elcomsoft Co. Ltd.
+PackageName: Advanced PDF Password Recovery
+License: Tiral
```
Comment (Trenly) [inline manifests/e/ElcomsoftCo/Ltd/AdvancedPDFPasswordRecovery/5.20.194.6989/ElcomsoftCo.Ltd.AdvancedPDFPasswordRecovery.locale.en-US.yaml:9]:
```suggestion
License: Trialware
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation failed with:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
> 'manifests\e\ElcomsoftCo\Ltd\AdvancedPDFPasswordRecovery\5.20.194.6989', expected 
> 'manifests\e\Elcomsoft\AdvancedPDFPasswordRecovery\5.20.194.6989'

The `Co` is missing from the path.

**Comment by upintheairsheep** [comment]
I really wish somebody with GIT desktop could save these pull requests for me.

**Comment by stephengillie** [comment]
> I really wish somebody with GIT desktop could save these pull requests for me.

I've found how with the GitHub web client. From the website, press dot "." to open the editor. On the left pane there's a file tree - right click the parent folder and click "New Folder", give it a name, then drag and drop these files there. With your permission I'd be happy to assist with your PRs.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**